differential-expression/gene_DE.Rmd

```diff
@@ -35,10 +35,10 @@ add them as data.frames to your environment.
 
 ```{r Import data from .tsv files}
 # Read in metadata tsv file
-metadata <- read.delim("../data/E-GEOD-39842_metadata.tsv", header = TRUE)
+metadata <- read.delim("../data/metadata_GSE71270.tsv", header = TRUE)
 
 # Read in data tsv file
-df <- read.delim("../data/E-GEOD-39842.tsv", header = TRUE, row.names = 1,
+df <- read.delim("../data/GSE71270.tsv", header = TRUE, row.names = 1,
```

Comment:
Would probably change both of these to use `file.path`

Reply:
Will do.